src/renderer/coremods/installer/commands.ts

```diff
@@ -0,0 +1,81 @@
+import { Injector } from "@replugged";
+import { ApplicationCommandOptionType } from "src/types";
+import { INSTALLER_SOURCES, InstallerSource, getInfo, install } from "./util";
+
+const injector = new Injector();
```

Comment:
Don't create a new injector here, just export the one from index.tsx and re-use it. Then you don't need to deal with uninjecting separately.

Reply:
Unless I extract the injector from index into a 'common' file, there would be dependency cycles or miss placed variables, which would be kind of a mess.

I think keeping a new injector here is the cleanest option?

Reply:
You could pass the injector as an arg to the function

Reply:
True, let me get on that